src/ontodia/customization/templates/VOWLClass.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { Component } from 'react';
+
+import { CrossOriginImage } from '../../viewUtils/crossOriginImage';
+
+import { TemplateProps } from '../props';
+
+const CLASS_NAME = 'ontodia-vowl-class-template';
+
+export class VowlClassTemplate extends Component<TemplateProps, {}> {
+    render() {
+        const {color, imgUrl, icon, types, label, isExpanded, iri, propsAsList} = this.props;
+        return (
+            <div className={CLASS_NAME}>
+                <div className={`${CLASS_NAME}__main-part`}>
+                        <div className='ontodia-vowl-class-template_body'>
+                        <div style={{flexGrow: 1, overflow: 'hidden', textOverflow: 'ellipsis'}}>
```

Comment:
Please, extract all inline styles for elements in markup for this template into corresponding `scss` file.

Reply:
done